sycl/test-e2e/Graph/Inputs/spec_constants_handler_api.cpp

```diff
@@ -0,0 +1,203 @@
+// This test is intended to check basic operations with SYCL 2020 specialization
+// constants using Graph and sycl::handler and sycl::kernel_handler APIs
+
```

Comment:
I'd mention in a comment here too that this test was taken from `SpecConstants/2020/handler-api.cpp`, and say if anything's changed.

Same for the other new test and `kernel-bundle-api.cpp`